### PR TITLE
add SRv6 uSID behaviors to the validonly lists

### PR DESCRIPTION
**Pull request by BYGX-wcr**
Add new SRv6 uSID behaviors to the validonly lists in several MY_SID entry attributes.

Signed-off-by: BYGX-wcr <wcr@live.cn>

### CODE REVIEW

File: inc/saisrv6.h [hunk 1]
```diff
@@ -373,7 +373,7 @@ typedef enum _sai_my_sid_entry_attr_t
      * @type sai_my_sid_entry_endpoint_behavior_flavor_t
      * @flags CREATE_AND_SET
      * @default SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_FLAVOR_NONE
-     * @validonly SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_E or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_X or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_T
+     * @validonly SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_E or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_X or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_T or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_uN or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_uA
```
Comment (r12f) [inline inc/saisrv6.h:376]: UN and UA might need to be upper case, e.g.: SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_UN

File: inc/saisrv6.h [hunk 2]
```diff
@@ -427,7 +427,7 @@ typedef enum _sai_my_sid_entry_attr_t
      * @objects SAI_OBJECT_TYPE_VIRTUAL_ROUTER
      * @allownull true
      * @default SAI_NULL_OBJECT_ID
-     * @validonly SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_T or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT4 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT6 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT46
+     * @validonly SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_T or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT4 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT6 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_DT46 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_uDT4 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_uDT6 or SAI_MY_SID_ENTRY_ATTR_ENDPOINT_BEHAVIOR == SAI_MY_SID_ENTRY_ENDPOINT_BEHAVIOR_uDT46
```
Comment (r12f) [inline inc/saisrv6.h:430]: UDT is the same, needs to be upper case.

**Comment by abdosi** [comment]
@BYGX-wcr : please address above comment. The semantics in comment should match the actual defiant ion

**Review by abdosi — CHANGES_REQUESTED** [review]
Please address community comments.

File: inc/saisrv6.h (hunk 1 above)
Comment (BYGX-wcr) [reply inc/saisrv6.h]: Sorry for my mistake. I have updated.

File: inc/saisrv6.h (hunk 2 above)
Comment (BYGX-wcr) [reply inc/saisrv6.h]: thanks

**Comment by kperumalbfn** [comment]
Changes LGTM

**Comment by kperumalbfn** [comment]
@yakivh Could you check

**Review by kperumalbfn — APPROVED** [review]

**Review by r12f — APPROVED** [review]

**Comment by r12f** [comment]
Thanks for the update!

**Review by kcudnik — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Review by ahsalam — APPROVED** [review]

**Review by cscarpitta — APPROVED** [review]

**Pull request merged**